docs/ASSESSMENTS.md

```
@@ -19,86 +20,90 @@ CodonCanvas assessments focus on **conceptual understanding** over technical mem
 - [ ] The genetic code
 
-**8. Mutation Types (Prior Knowledge)**
-Have you heard of these mutation types? (Check all you know)
+**8. Mutation Types (Prior Knowledge)**\
+Have you heard of these mutation types?\
+(Check all you know)
 
 - [ ] Silent mutation
 - [ ] Missense mutation
 - [ ] Nonsense mutation
 - [ ] Frameshift mutation
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)